lib/Pipeline.js

```diff
@@ -82,14 +83,22 @@ Pipeline.processJSON = function(gltf, options) {
  */
 Pipeline.processJSONWithExtras  = function(gltfWithExtras, options) {
     options = defaultValue(options, {});
+
+    addDefaults(gltfWithExtras, options);
+
+    // Generate stats for unoptimized input
+    if (options.stats) {
+        gltfStatistics.printStatistics(gltfWithExtras, options.inputPath);
+    }
```

Comment:
I looked at this more carefully and noticed that `addDefaults` doesn't add any items that are used by `gltfStatistics`. It does add techniques, shaders, programs and other things when the model materials common extension is used. The order here is ok for now though.